### PR TITLE
[UI Enhancements]: Local Storage, Responsive CSS, and Response Regeneration

### PR DESCRIPTION
**Pull request by AdityaP700**
**Overview:**
This pull request refers to issue #146 which introduces several enhancements to the project, focusing on user experience and responsiveness. 
### **Changes Made:**
1. **Local Storage Implementation:** 
   - Added local storage functionality to save user inputs, improving data retention across sessions.
2. **Responsive Design Adjustments:** 
   - Updated CSS styles to enhance the responsiveness of the website, ensuring optimal viewing on various devices.
3. **Regenerative Response Button:**
   - Implemented a new button that allows users to regenerate responses seamlessly, enhancing interaction with the application.

https://github.com/user-attachments/assets/59122930-690b-4cc9-803e-de5bd5bfe185

### CODE REVIEW

**Comment by AdityaP700** [comment]
Concerned Project Admin, please refer to my PR for issue #146 and  if  it's possible kindly merge this PR under any Level of GSSOC-extd. Thanks in Anticipation.

**Review by 1Shubham7 — CHANGES_REQUESTED** [review]
amazing work @AdityaP700, these are really meaningful  changes. There is one thing we need to reconsider, we would want a guy to only get one wife, if he regenerates , he should go back to the part where name is entered. what do you say : )

**Comment by 1Shubham7** [comment]
your PR title, description everything is so well done, amazing brother, keep this up while contributing to larger projects too : )

**Comment by AdityaP700** [comment]
Hello @1Shubham7 ,thanks for your appreciation . Well i did had a thought of considering your point but since i was assigned to fix my issue so i simply made the changes accordingly . But since your suggestion seems legit ,i would surely update the codebase soon and raise the PR . Again Thanks for your kind Words :)

**Comment by AdityaP700** [comment]
Hello @1Shubham7  ,Apologies for the delay. I was addressing a few other issues, but everything should be in order now. Please take a look at the latest changes when you have a chance. If you have any feedback, suggestions, or notice any issues, feel free to let me know. Looking forward for your input :)

**Comment by 1Shubham7** [comment]
hey @AdityaP700, you are a bit late, I merged someone's PR about response regeneration (we call it "Play again") . but this PR is really meaningful, I would request you to remove the response regeneration part from the PR and it is a good PR for local storage and CSS : )

**Review by 1Shubham7 — CHANGES_REQUESTED** [review]
hey @AdityaP700, you are a bit late, I merged someone's PR about response regeneration (we call it "Play again") . but this PR is really meaningful, I would request you to remove the response regeneration part from the PR and it is a good PR for local storage and CSS : )

**Comment by 1Shubham7** [comment]
once you are done with changes, please change a video of how the website looks after that : )

File: src/components/Confirm.js [hunk 1]
```diff
@@ -66,13 +103,6 @@ function ConfirmationCheckbox() {
     setPerson(wives[randomNum]);
   };
 
-  const playAgain = () => {
```
Comment (1Shubham7) [inline src/components/Confirm.js:69]: why are you removing the playAgain button? please don't remove it.

**Review by 1Shubham7 — CHANGES_REQUESTED** [review]
please don't remove the playAgain button, make those changes : )

File: src/components/Confirm.js (hunk 1 above)
Comment (AdityaP700) [reply src/components/Confirm.js:69]: Apology for removing it ,will be adding it back with all functionality for sure

**Comment by AdityaP700** [comment]
Hello @1Shubham7,Apologies for the delays and any confusion caused. As a beginner, this was one of my first issues, and I’ve learned a lot throughout the process—whether it’s using GitHub effectively, implementing React hooks, or performing responsive checks.
In the latest commit, I’ve re-added a Play Again button allowing users to restart the game. Additionally, I’ve fixed a minor bug in the My.css file, improving the overall styling .
https://github.com/user-attachments/assets/31122cdc-ef54-4ac5-90a0-d0735ec06318


I’ve made my latest commit, and I would appreciate it if you could kindly review it.

**Comment by 1Shubham7** [comment]
> Hello @1Shubham7,Apologies for the delays and any confusion caused. As a beginner, this was one of my first issues, and I’ve learned a lot throughout the process—whether it’s using GitHub effectively, implementing React hooks, or performing responsive checks. In the latest commit, I’ve re-added a Play Again button allowing users to restart the game. Additionally, I’ve fixed a minor bug in the My.css file, improving the overall styling . https://github.com/user-attachments/assets/31122cdc-ef54-4ac5-90a0-d0735ec06318
> 
> I’ve made my latest commit, and I would appreciate it if you could kindly review it.

You have done amazing work brother, keep it up! thanks a lot for your contribution : ) I think can be merged now : )

**Comment by AdityaP700** [comment]
Hey there @1Shubham7 ,Well I have noticed that the PR which have been merged hasn't got updated  in the leaderboard yet despite having the necessity Labels . Well I was suggested by the concerned admins to contact you regarding this so ,kindly look into this so as my PR can be reflected in the Leaderboard. Thanks in anticipation:)

**Pull request merged**